frontend/javascripts/oxalis/model/reducers/volumetracing_reducer_helpers.ts

```diff
@@ -179,6 +182,23 @@ export function setMappingNameReducer(
   if (mappingType !== "HDF5" || !isMappingEnabled) {
     mappingName = null;
   }
+  if (mappingName) {
+    const { agglomerates, mappings } = getSegmentationLayerByName(
+      state.dataset,
+      volumeTracing.tracingId,
+    );
+    const listToSearchIn = mappingType === "HDF5" ? agglomerates : mappings;
+    if (
+      mappingType === "HDF5" &&
+      (listToSearchIn == null || listToSearchIn.indexOf(mappingName) === -1)
+    ) {
+      console.error(
+        `Cannot set mapping ${mappingName} of type ${mappingType} for volume tracing ${volumeTracing.tracingId} because it is not included in the mapping list of the data layer: ${listToSearchIn}.`,
+      );
+      return state;
+    }
+  }
+
```

Comment:
Ok I did some more testing regarding this error handling. Turns out, when a default mapping is configured, this code is executed before the existing mappings for each layer are loaded by the frontend to set the inital mapping. Thus, this check here would cause the error being logged and then the inital mapping settings would be ignored. Therefore, I removed this error catching mechanism here for now.

To re-add this, I think we need to refactor the loading mechanism of initial mappings. 

@daniel-wer what do you think about this? I think to fix the error of using the correct volume tracing is for now a good enough improvement and should be merged better early than late